### DISCUSSION TITLE
Macbook pro 16" 2019 i7 5300M supported?

### DISCUSSION THREAD

**Discussion by shamalawy** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#302] [Q&A] [answered]
I was about to buy the pro license when it mentioned that it only supports the macbook pro 16 with 5600M graphics card.

Does the pro license support the one with 5300M graphics card?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @shamalawy,

AlDente Pro supports every version of the 16" MacBook Pro.

**Reply by shamalawy** [reply]
Thank you
